src/Atoms/Card.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CardWrapper = styled.div`
```

Comment:
Name of a file is different than name of function component. 
Also maybe that component should be in Styles folder, as he has only styles? if it is his future puropse.
And maybe there is better word than Card as name? (for what exactly is that wrapper?)

Reply:
This wrapper can be used as a wrapper for many components, i was thinking about it as a wrapper for views. Placement of the file could be in Templates or Styles its up to You. 
Name of the component itself is Card.   CardWrapper is just a name of styled component.
For me Card is ok, but of course we can change to Wrapper perhaps?

Reply:
Maybe Wrapper, or CardWrapper? Just in case there will be more wrappers. 
If it will give styles for many components, I think, lets put it to styles. 
So for ex. template+wrapper will be a page